extension/src/openvic-extension/classes/GUIScrollbar.cpp

```diff
@@ -369,37 +363,43 @@ Error GUIScrollbar::set_gui_scrollbar(GUI::Scrollbar const* new_gui_scrollbar) {
 
 	/* _Element is either GUI::Button or GUI::Icon, both of which have their own
 	 * separately defined get_sprite(), hence the need for a template. */
-	const auto set_texture = [&gui_scrollbar_name]<typename _Element>(
-		String const& target, _Element const* element, Ref<GFXSpriteTexture>& texture
-	) -> bool {
-		ERR_FAIL_NULL_V_MSG(element, false, vformat(
-			"Invalid %s element for GUIScrollbar %s - null!", target, gui_scrollbar_name
-		));
+	const auto set_texture =
+		[&gui_scrollbar_name]<typename _Element>(String const& target, _Element const* element, Ref<GFXSpriteTexture>& texture)
+		-> bool {
+		ERR_FAIL_NULL_V_MSG(element, false, vformat("Invalid %s element for GUIScrollbar %s - null!", target, gui_scrollbar_name));
 		const String element_name = Utilities::std_to_godot_string(element->get_name());
```

Comment:
IMO the lambda function's body should be indented one level further than it's capture/arguments section, and I would prefer not have a line break in the middle of `) -> bool {`

Reply:
There is literally no formatter capable of that because that lambda definition doesn't reside in an indentation.